cluster/charts/rook-ceph-cluster/templates/_helpers.tpl

```diff
@@ -4,3 +4,41 @@ Define the clusterName as defaulting to the release namespace
 {{- define "clusterName" -}}
 {{ .Values.clusterName | default .Release.Namespace }}
 {{- end -}}
+
+{{/*
+Return the target Kubernetes version
+*/}}
+{{- define "capabilities.kubeVersion" -}}
+{{- if .Values.global }}
+    {{- if .Values.global.kubeVersion }}
+    {{- .Values.global.kubeVersion -}}
+    {{- else }}
+    {{- default .Capabilities.KubeVersion.Version .Values.kubeVersion -}}
+    {{- end -}}
+{{- else }}
+{{- default .Capabilities.KubeVersion.Version .Values.kubeVersion -}}
+{{- end -}}
+{{- end -}}
```

Comment:
Why would a user want to change the kube version away from what Helm detects? This seems like a code smell to me.

Reply:
So my use case is a little different to a normal helm user, I use argocd to deploy helm charts, and quite often you'll see things like this in very widely used opensource helm charts to allow more configurability.

For my specific usage, it is useful to be able to force the helm renderer inside argocd to target a given version of kubernetes as capabilities.KubeVersion.Version may not hold the value you expect.

More examples from other charts:
* [prometheus open source helm charts](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/templates/_helpers.tpl#L132-L146) 
* [bitnami helm templating](https://github.com/bitnami/charts/blob/master/bitnami/common/templates/_capabilities.tpl#L3-L16) 
* [from argocd](https://github.com/argoproj/argo-helm/blob/master/charts/argo-cd/templates/_helpers.tpl#L155-L175) 

I'm happy to change it to something more simple, but figured I would propose the more "helm" way of fixing it.

Reply:
I've simplified this section in the _helpers.tpl, as we don't need the double layer of configurability it had before